docs/source/_topic_map.yml

```diff
@@ -14,8 +14,8 @@ Topics:
         File: installing
       - Name: Quickstart
         File: quickstart
-      - Name: Installing Docker Machine drivers
-        File: docker-machine-drivers
+      - Name: Setting up the hypervisor driver plugin
+        File: setting-up-hypervisor-driver-plugin
```

Comment:
I think I got it now after reading overview of "Setting up the hypervisor driver plugin" doc. Make sense IMO :+1: